internal/app/secretless/providers/vault/provider.go

```
@@ -27,16 +27,16 @@ func ProviderFactory(options plugin_v1.ProviderOptions) plugin_v1.Provider {
 		log.Panicf("ERROR: Could not create Vault provider: %s", err)
```

this should throw an error too